### PR TITLE
feat(PageSingleFeature): Allow image to act as a link

### PR DESCRIPTION
**Pull request by vonbarown**
### Overview

Updating the image component the ``PageSingleFeature`` to act as a a link if provided with a url.

### PR Checklist

- [ ] Related to designs:
- [x] Related to JIRA ticket:  [EN-1895](https://codecademy.atlassian.net/browse/EN-1895)
- [x] I have run this code to verify it works
- [x] This PR includes unit tests for the code change
- [x] This PR includes testing instructions tests for the code change
- [ ] The alpha package of this PR is passing end-to-end tests in all relevant Codecademy repositories

#### Testing Instructions

1. Visit https://tengu.codecademy.com/api/portal/preview?path=pages/business&PR_ENV=portal-app-pr-2236
2. scroll to the "Close the gaps" gap section and click on the image
3. ensure that it navigate you to https://trial.codecademy.com/en-us/motleyfoolcasestudy?source=blp
4. Visit https://tengu.codecademy.com/pages/career-center?PR_ENV=portal-app-pr-2236
5. scroll to the "Do it all with Codecademy Pro" and ensure that you can't click on the image

[Don't make me tap the sign.](https://i.imgur.com/sy93D9I.png)

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](https://github.com/codecademy-engineering/Codecademy/pull/31505)  | [Monolith Env](http://www.google.fr/ 'Named link title') |
| Portal       | [Portal Link](https://github.com/codecademy-engineering/portal-app/pull/2206)  | [Portal Env](http://localhost:3100/pages/business)   |

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.153-alpha.f876e1.0<br/>@codecademy/gamut-labs@32.4.1-alpha.f876e1.0<br/>@codecademy/styleguide@58.1.8-alpha.f876e1.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://62c5829ac28dad1dacb48ca3--gamut-preview.netlify.app>https://62c5829ac28dad1dacb48ca3--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/62c5829ac28dad1dacb48ca3>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.153-alpha.d59ef5.0<br/>@codecademy/gamut-labs@32.4.1-alpha.d59ef5.0<br/>@codecademy/styleguide@58.1.8-alpha.d59ef5.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://62c5bc06abe5d803727287e6--gamut-preview.netlify.app>https://62c5bc06abe5d803727287e6--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/62c5bc06abe5d803727287e6>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.156-alpha.094012.0<br/>@codecademy/gamut-labs@32.6.1-alpha.094012.0<br/>@codecademy/styleguide@58.1.11-alpha.094012.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://62cc1e13e304240608f858b6--gamut-preview.netlify.app>https://62cc1e13e304240608f858b6--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/62cc1e13e304240608f858b6>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.156-alpha.92f591.0<br/>@codecademy/gamut-labs@32.6.1-alpha.92f591.0<br/>@codecademy/styleguide@58.1.11-alpha.92f591.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://62cc21da08f44e095b672817--gamut-preview.netlify.app>https://62cc21da08f44e095b672817--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/62cc21da08f44e095b672817>Deploy Logs</a></p>

**Comment by dreamwasp** [comment]
@vonbarown please fill out the [testing instructions](https://i.imgur.com/sy93D9I.png)

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.156-alpha.b26a74.0<br/>@codecademy/gamut-labs@32.6.1-alpha.b26a74.0<br/>@codecademy/styleguide@58.1.11-alpha.b26a74.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://62cc3a5bbf3a862331c7df78--gamut-preview.netlify.app>https://62cc3a5bbf3a862331c7df78--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/62cc3a5bbf3a862331c7df78>Deploy Logs</a></p>

**Comment by aresnik11** [comment]
@vonbarown fun fact! instead of changing the code to add `preview: true` you can just go to http://localhost:3100/api/portal/preview?path=pages/business and that enables preview mode. you'll be able to tell because there's an alert banner telling you this is contentful preview mode

**Comment by vonbarown** [comment]
> @vonbarown fun fact! instead of changing the code to add `preview: true` you can just go to http://localhost:3100/api/portal/preview?path=pages/business and that enables preview mode. you'll be able to tell because there's an alert banner telling you this is contentful preview mode

Thanks, save a lot of head 😆

File: packages/gamut-labs/src/LandingPage/PageSingleFeature.tsx [hunk 1]
```diff
@@ -36,6 +38,33 @@ type Eyebrow = {
...
+  return (
+    <Container
+      display={{
+        _: hideImageOnMobile ? 'none' : 'initial',
+        sm: 'initial',
+      }}
+      href={linkUrl}
+      width={1}
+      tabIndex={linkUrl ? 0 : undefined}
+    >
```
Comment (dreamwasp) [inline packages/gamut-labs/src/LandingPage/PageSingleFeature.tsx:62]: did some[ brief research](https://usability.yale.edu/web-accessibility/articles/images) and images as links should probably require alt text, maybe worth tagging @Codecademy/accessibility and/or posting in the accessibility slack channel

File: packages/styleguide/stories/Brand Labs/Organisms/PageSingleFeature.stories.mdx [hunk 2]
```diff
@@ -193,3 +193,26 @@ When provided a gif a CTA to allow users to play/pause the gif will be displayed
     )}
   </Story>
 </Canvas>
+
+## Linakble Image
```
Comment (dreamwasp) [inline packages/styleguide/stories/Brand Labs/Organisms/PageSingleFeature.stories.mdx:197]: *Linkable

**Review by dreamwasp — CHANGES_REQUESTED** [review]
just wanna make sure this is accessible before ✅

File: packages/gamut-labs/src/LandingPage/PageSingleFeature.tsx (hunk 1 above)
Comment (vonbarown) [reply packages/gamut-labs/src/LandingPage/PageSingleFeature.tsx:62]: should the link have the alt text as well?  This component has the alt text as required

Comment (vonbarown) [reply packages/gamut-labs/src/LandingPage/PageSingleFeature.tsx:62]: When running this through voice over, it announces that it is a link and an image with the alt tag
Screen Shot 2022-07-12 at 11 45 04 AM

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.156-alpha.be19c8.0<br/>@codecademy/gamut-labs@32.6.1-alpha.be19c8.0<br/>@codecademy/styleguide@58.1.11-alpha.be19c8.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://62cee03f1f72a733359b61b4--gamut-preview.netlify.app>https://62cee03f1f72a733359b61b4--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/62cee03f1f72a733359b61b4>Deploy Logs</a></p>

**Review by dreamwasp — APPROVED** [review]
nice!

**Pull request merged**